ethcore/src/account_provider/no_hw.rs

```diff
@@ -0,0 +1,42 @@
+//! Dummy module for platforms that does not provide support for hardware wallets (libusb)
+
+use super::{fmt, Address};
+
+pub struct WalletInfo {
```

Comment:
copy-paste of `hardware_wallet::WalletInfo`

Reply:
Couldn't this be a newtype of that?